_episodes/03-intro-git-github.md

```diff
@@ -1474,14 +1476,40 @@ origin   https://github.com/USERNAME/un-report.git (fetch)
 ```
 {: .output}
 
-Once the remote is set up, this command will push the changes from
-our local repository to the repository on GitHub:
+Now we want to send our local git information to GitHub. While the default for code you put on GitHub is that anyone can view or make copies of your code, in order to make changes to your repository, you need to be able to log in so GitHub can recognize you as someone who is authorized to make changes.
+
+### Setting up your GitHub Personal Access Token (PAT)
+
+When you use the GitHub website, you need to login with a username and password. By default, only you will be able to make any changes to the repositories you create. In order to perform git commands on your own computer that interact with GitHub, we need a way to tell GitHub who you are. Rather than requiring you to type your password every time, you can create identify yourself with a personal access token (PAT). Let's first tell git that we would like it to remember our credentials so we don't have to constantly retype them. At the command line type:
+
+```
+git config --global credential.helper store
+```
+{: .language-bash}
+
+Like the previous `git config` commands we ran before, this tells git to store our account information so it doesn’t have to ask us for it every time time we use git on the command line.
+
+The information git stores is your personal access token (PAT). These tokens are basically a secret word that only you know that allows you to access all your stuff. Think of these tokens like a key to your house. You never want to hand over the keys to your house to someone you don't trust. But as long as you hang on to that key, you are free to access all your stuff.
+
+To create a PAT, you'll need to be logged into to GitHub. Click your profile icon on the top left and choose "Setting" from the dropdown. On the main settings page there is a long list of options on the left. Scroll down till you find "Developer Settings". Next you should see three options: "GitHub Apps", "OAuth Apps", and "Personal access tokens". We want to create a token so click on the last link. You should now see a link to " Generate a personal access token". Click that. (You should now be at https://github.com/settings/tokens/new)
+
+On the "New personal access token" form, the first field you see is for "Note." You can actually create multiple tokens. This note field helps you remember what the token was for. It's a good idea to create one per computer you use so the note field would be something like "work-laptop", "home-macbook", or "greatlakes-project". Next you will see an option for "Expiration." Since your tokens are like the keys to your house, it might be a good idea that if you forgot about your tokens, they just stop working after a while so no one else can misuse them. When your tokens expire, you can just generate a new one. GitHub recommends you choose an expiration date so we can just choose "90 days" or whatever is appropriate for you.
```

Comment:
May want to mention at the end that they'll have to do this process again after their token expires

Reply:
Thanks. I added this note as well.